hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2244,6 +2248,14 @@ public OmKeyInfo lookupKey(OmKeyArgs args) throws IOException {
     }
   }
 
+
+  @Override
+  public void renameKeys(OmRenameKeys omRenameKeys)
```

Comment:
RenameKey on Bucket seems also not related with this JIRA if I understand this PR correctly.

Reply:
NA, like above.